Sources/TokamakCore/Styles/PickerStyle.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public protocol PickerStyle {}
+
+public struct PopUpButtonPickerStyle: PickerStyle {}
+
+public struct RadioGroupPickerStyle: PickerStyle {}
+
+public struct DefaultPickerStyle: PickerStyle {}
```

Comment:
I think you have to re-export these in `TokamakDOM`.

Reply:
I don't think so, there are no corresponding types for these in SwiftUI and users can change their style with the `pickerStyle` modifier, which they have access to.

Reply:
I.e. users don't need to access these types directly, they just use the `Picker` type and apply the modifier on it when needed.

Reply:
Oh, my bad, I thought you were referring to the underscored view types. You're totally right, will re-export them.